docs/pages/access-controls/guides/webauthn.mdx

```diff
@@ -138,38 +174,72 @@ $ tsh login --proxy=example.com
 #   Extensions:         permit-agent-forwarding, permit-port-forwarding, permit-pty
 ```
 
+</ScopedBlock>
+<ScopedBlock scope={["cloud"]}>
+
+```code
+$ tsh login --proxy=mytenant.teleport.sh
+# Enter password for Teleport user codingllama:
+# Tap any security key or enter a code from a OTP device:
+# > Profile URL:        https://mytenant.teleport.sh
+#   Logged in as:       codingllama
+#   Cluster:            mytenant.teleport.sh
+#   Roles:              access, editor
+#   Logins:             codingllama
+#   Kubernetes:         enabled
+#   Valid until:        2021-10-04 23:32:29 -0700 PDT [valid for 12h0m0s]
+#   Extensions:         permit-agent-forwarding, permit-port-forwarding, permit-pty
+```
+
+</ScopedBlock>
+
 <Admonition type="note">
-  WebAuthn for logging into Teleport is only required for [local users](
+  WebAuthn for logging in to Teleport is only required for [local users](
   ../../setup/reference/authentication.mdx#local-no-authentication-connector). SSO users should configure
   multi-factor authentication in their SSO provider.
 </Admonition>
 
 ## Migrating from U2F
```

Comment:
We can probably remove this section from master (10.0) docs, since U2F is not supported.

I would keep your changes in v9 and v8 because migrating from U2F to Webauthn is still something you might need to do on those versions.